src/Tests/AzureClientTests.cs

```diff
@@ -180,7 +179,9 @@ public void TestAllTargets()
             // connect to mock workspace with all providers
             var targets = ExpectSuccess<IEnumerable<TargetStatusInfo>>(ConnectToWorkspaceAsync(azureClient, MockAzureWorkspace.NameWithMockProviders));
             // 2 targets per provider: mock and simulator.
-            Assert.AreEqual(2 * Enum.GetNames(typeof(AzureProvider)).Length, targets.Count());
+
+            // TODO: uncomment once there is a microsoft.mock target
+            //Assert.AreEqual(2 * Enum.GetNames(typeof(AzureProvider)).Length, targets.Count());
```

Comment:
Will the addition of a microsoft.mock target be addressed as a separate PR? If so, this TODO should also be tracked somewhere so we don't forget.

Reply:
When I raised the issue, it didn't seem to be a priority with anyone. I can make a git issue for it nevertheless. I currently have no plans to execute on this, and if I did I would need to have someone more experienced walk me through how to add `microsoft.mock`, as I wasn't able to figure it out on my own.